### PR TITLE
DOC: Release notes for v7.2.0

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

It is time for a pcdsdevices tag. This is a really big tag.
It is ok to follow this up with a smaller tag if needed (if some of the other PRs finish in the next few days, for example).

Reviewers are everyone who contributed that github lets me ask for a review. Thank you for contributing. You are not obligated to review, but if I've misrepresented your contribution please correct me.

## Motivation and Context

I want to tag every package that I help maintain since we haven't done this at all in 2023.

## How Has This Been Tested?

n/a

## Where Has This Been Documented?

this is docs

### CODE REVIEW

**Review by tangkong — APPROVED** [review]
lots of good work here 👍

File: docs/source/releases.rst [hunk 1]
```diff
@@ -2,6 +2,96 @@ Release History
...
+Features
+--------
+- Added a ``diff_configuration`` function to the ``IMS`` class in
+  ``epics_motor.py`` that compares the desired motor pv settings with the
+  current configuration assigned in the parameter manager.
+- Made ``IMS`` pmgr only search through ``USR`` objects.
+
+Device Updates
+--------------
+- Adds a preliminary attenuator class `AT1K2` and base classes for similar
```
[the remark this answers is not in the document]
Comment (ZLLentz) [reply docs/source/releases.rst:17]: Default role is set to `any`, so single backtick hyperlinks to the corresponding API page. I'll build the docs locally and make sure these aren't broken links.

[the remark this answers is not in the document]
Comment (ZLLentz) [reply docs/source/releases.rst:17]: Some of these links work and others are broken. I'm adjusting accordingly.

[the remark this answers is not in the document]
Comment (tangkong) [reply docs/source/releases.rst:17]: So is it single backticks for linking classes to docs and double backticks for literal/monospace text?

**Review by nrwslac — APPROVED** [review]
lgtm! 🚀

File: docs/source/releases.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (ZLLentz) [reply docs/source/releases.rst:17]: Here's the sphinx docs that remember much better than I do:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html

tl;dr you can do:
```
:any:`KeyWord`
```
to make a link to wherever and whatever ``KeyWord`` is in your doc.
There may be a more precise role to pick for the link depending on the situation.

If you set `default_role` in conf.py to `any`, then you can just do
```
`KeyWord`
``` 
To make the link.

And in pcdsdevices we have a big set of API pages that have all of the class names as possible link targets.
